bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/SortedLedgerStorage.java

```diff
@@ -354,4 +355,10 @@ public boolean isInForceGC() {
     public List<GarbageCollectionStatus> getGarbageCollectionStatus() {
         return interleavedLedgerStorage.getGarbageCollectionStatus();
     }
+
+    @Override
+    public byte[] getEntriesOfLedger(long ledgerId) throws Exception {
```

Comment:
See: this shouldn't return a byte[] -- it should return an iterator representing the merged streams from the memtable and interleavedLedgerStorage.  The caller should be responsible for encoding it before it gets sent back.

Reply:
as mentioned in other comment - 

https://github.com/apache/bookkeeper/pull/1931/files/a9d00a25d29bd034c7ee070f7a0d75bac29a94d2#r254091827

Reply:
new logic should take care of this